tests/devices_test/device_properties_test.py

```diff
@@ -79,10 +80,8 @@ def __init__(self, methodName='run_test'):
             "path": xform(lambda x, m: six.assertRegex(self, x, "^/dev", m)),
             "raw_device": xform(self.assertIsNotNone),
             "resizable": xform(self.assertFalse),
-            "size": xform(lambda x, m: self.assertEqual(x, Size(0), m)),
             "status": xform(self.assertFalse),
             "sysfs_path": xform(lambda x, m: self.assertEqual(x, "", m)),
-            "target_size": xform(lambda x, m: self.assertEqual(x, Size(0), m)),
```

Comment:
Why did you have to remove these from the base class and add them to the various subclasses?

Reply:
Fixed: Moved the lines back to base class. Only those with non-default values were left in subclasses.